### PR TITLE
fixes unselectable sk'akh robes

### LINKED ISSUE #18652: [Bug]: Unable to select between Sk'akh priest robes in loadout
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Horizon

### Describe the issue

Selecting the type for the new Sk'akh priest robes in the Unathi loadout does not open the window to select the other options, meaning you can only take the default priest's robes.

### How to reproduce

_No response_

### Round ID

csI-a9sn

### Staff ckey

_No response_

### PR DESCRIPTION
**Pull request by RustingWithYou**
Fixes #18652

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!bugfix

**Comment by RustingWithYou** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/items/xeno/unathi.dm [hunk 1]
```diff
@@ -458,19 +458,20 @@
 	cards["ouerea passcard"] = /obj/item/clothing/accessory/badge/passcard/ouerea
 	gear_tweaks += new /datum/gear_tweak/path(cards)
 
-/datum/gear/uniform/skakh
-	display_name = "sk'akh robes selection"
+/datum/gear/uniform/unathi/skakh
+	display_name = "Skakh robes selection"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/items/xeno/unathi.dm:462]:
```suggestion
	display_name = "ska'kh robes selection"
```

File: code/modules/client/preference_setup/loadout/items/xeno/unathi.dm [hunk 2]
```diff
@@ -458,19 +458,20 @@
...
 	path = /obj/item/clothing/under/unathi/skakh
 	cost = 1
 	whitelisted = list(SPECIES_UNATHI)
 	sort_category = "Xenowear - Unathi"
 	allowed_roles = list("Chaplain")
+	flags = GEAR_HAS_NAME_SELECTION|GEAR_HAS_DESC_SELECTION
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/items/xeno/unathi.dm:468]:
```suggestion
	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
```

**Pull request merged**